### PR TITLE
Feature/development phase 3

### PR DESCRIPTION
**Pull request by skyv26**
# Achievement `#3` : To Do interactive list

I improved my todo list by adding more interactions and resolve some bugs in order to maintain a good User Experience. I also tried to make it dynamic.

- Added new JavaScript file and import it as a module.
- Added method for status updates
- Added event listener to the checkbox
- Updated items object's value for `completed` key upon user actions.
- Added the **list** `placeholders` inside the `index.html` to hold the todo-task but using `pure-javascript`.
- Implemented function for the `**Clear all completed**` button.
- Added separate `scss` for different component and render the style with the help of `scss-loader`, `css-loader` and `style-loader`.
- Added list **`SORTABLE`** functionality.
- Todo list rendered according to index value.
- Finally, Files generated with `webpack` using `npm run build` and served from `dist/` folder using `webpack-dev-serve` command.
 - Resolved the linters error for `html`, `css/scss` and `javascript`.

### CODE REVIEW

File: src/components/main/Operation.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import './Operation.scss';
+
+import Button from '../elements/Button';
+import Wrapper from '../elements/Wrapper';
+import TodoLib from '../../store/TodoLib';
+
+const Operation = () => {
```
Comment (Ismailco) [inline src/components/main/Operation.js:8]: - [x] When removing completed tasks, the index is not correct after removing some items from the list, kindly make sure it updates all the items indexes so the updated array of items is stored in local storage with the correct list values after the page reloads

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @skyv26, 

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/main/Operation.js (hunk 1 above)
Comment (skyv26) [reply src/components/main/Operation.js:8]: @Ismailco Ok Thank You.

**Review by Ismailco — APPROVED** [review]
### Hi @skyv26, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

cr-approve


## Highlights

✔️  No linter errors
✔️ Used Github Flow
✔️ Work documented professionally

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by skyv26** [comment]
@Ismailco Thank you for your valuable suggestion and feedback.

Happy Coding ! 🥂

**Pull request merged**